saw/llvm.py

```diff
@@ -320,7 +322,9 @@ def fresh_var(self, type : LLVMType, suggested_name : Optional[str] = None) -> F
             raise Exception("wrong state")
         return v
 
-    def alloc(self, type : LLVMType, *, read_only : bool = False, points_to : Optional[SetupVal] = None) -> SetupVal:
+    def alloc(self, type : LLVMType, *, read_only : bool = False,
+                                        alignment : Optional[int] = None,
```

Comment:
There are docstrings for the other keyword params -- are there a few words for `alignment` to help out users?

Reply:
Oops, I completely forgot about documentation. Fixed.